### PR TITLE
eol: add page

### PR DESCRIPTION
**Pull request by adriens**
# :heavy_check_mark: Prerequisites

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] Version of the command being documented (if known):**

# :bookmark: Related resources

- [⌛ Manage EoLs like a boss with endoflife.date 🛑 ](https://dev.to/optnc/manage-eols-like-a-boss-with-endoflifedate-2ikf)

### CODE REVIEW

**Comment by adriens** [comment]
:partying_face:

File: pages/common/eol.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# eol
+
+> CLI to show end-of-life (EOLs) dates for an (ever increasing) number of products (Programming languages, Devices, databases, Operating Systems, Frameworks, Desktop Applications, Application server, Cloud Services and other categories.
```
Comment (pixelcmtd) [inline pages/common/eol.md:3]:
```suggestion
> CLI to show end-of-life dates (EOLs) for an (ever increasing) number of products (e.g. Programming Languages, Frameworks, Devices, Operating Systems, Desktop Applications, ...)
> More information: <https://github.com/hugovk/norwegianblue>.
```

**Comment by adriens** [comment]
:crossed_fingers:

**Comment by 4G3NT** [comment]
You should read CONTRIBUTING.md and use `tldr -f [file]` to preview your tldr page. Additionally, the commit messages you have used do not follow the convention. You need amend the commits and perform a force push.

**Comment by kbdharun** [comment]
> You should read CONTRIBUTING.md and use `tldr -f [file]` to preview your tldr page. Additionally, the commit messages you have used do not follow the convention. You can amend the commits and perform a force push.

We generally don't recommend force-pushing in tldr as it makes any unresolved reviews outdated. 

Regarding the commit messages, it is fine if the first commit/PR name follows the convention, as it is what will be visible upon Squash and Merge. If there are a lot of minor commits we will clean the commit message upon merge (having only the coauthors present).

**Comment by Waples** [comment]
I'd rather see this back as a clean PR than the state its currently in.

File: pages/common/eol.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`eol {{product}} -f markdown`
+
+- Get eols of a product as csv:
+
+`eol {{product}} -f csv`
+
+- Get eols of a multiple products as a single markdown file:
+
+`eol {{product1}} {{product2}} {{product3}} -f markdown > eol_report.md`
```
Comment (kbdharun) [inline pages/common/eol.md:36]:
```suggestion
> CLI to show end-of-life dates (EoLs) for a number of products.
> More information: <https://github.com/hugovk/norwegianblue>.

- List all available products:

`eol`

- Get EOLs of a given product:

`eol {{product}}`

- Open product webpage on EOL:

`eol {{product}} -w`

- Get EOLs of multiple products all at once:

`eol {{product1 product2 ...}}`

- Get EOLs of a product in a specific format:

`eol {{product}} -f {{html|json|md|markdown|pretty|rst|csv|tsv|yaml}}`

- Get EOLs of multiple products as a single markdown file:

`eol {{product1 product2 ...}} -f markdown > eol_report.md`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr. I have some suggestions for this page.

**Comment by adriens** [comment]
;-( @kbdharun

File: pages/common/eol.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+> CLI to show end-of-life dates (EoLs) for a number of products.
+> More information: <https://github.com/hugovk/norwegianblue>.
+
+- List all available products:
+
+`eol`
+
+- Get EOLs of a given product:
+
+`eol {{product}}`
```
Comment (acuteenvy) [inline pages/common/eol.md:10]:
```suggestion
- Get EoLs of one or more products:

`eol {{product1 product2 ...}}`
```

File: pages/common/eol.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+- Open product webpage on EOL:
+
+`eol {{product}} -w`
+
+- Get EOLs of multiple products all at once:
+
+`eol {{product1 product2 ...}}`
+
```
Comment (acuteenvy) [inline pages/common/eol.md:19]: Merged with the first example.
```suggestion
```

File: pages/common/eol.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- Get EOLs of a product in a specific format:
+
+`eol {{product}} -f {{html|json|md|markdown|pretty|rst|csv|tsv|yaml}}`
```
Comment (acuteenvy) [inline pages/common/eol.md:22]:
```suggestion
- Get EoLs of a one or more products in a specific format:

`eol {{product1 product2 ...}} --format {{html|json|md|markdown|pretty|rst|csv|tsv|yaml}}`
```

File: pages/common/eol.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+- Get EOLs of multiple products as a single markdown file:
+
+`eol {{product1 product2 ...}} -f markdown > eol_report.md`
```
Comment (acuteenvy) [inline pages/common/eol.md:26]:
```suggestion
- Get EoLs of one or more products as a single markdown file:

`eol {{product1 product2 ...}} --format {{markdown}} > {{eol_report.md}}`
```

File: pages/common/eol.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+> CLI to show end-of-life dates (EoLs) for a number of products.
```
Comment (acuteenvy) [inline pages/common/eol.md:1]: @kbdharun you accidentally removed the title

```suggestion
# eol

> CLI to show end-of-life dates (EoLs) for a number of products.
```

**Review by acuteenvy — COMMENTED** [review]
Long options: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax and other suggestions.

File: pages/common/eol.md (hunk 7 above)
Comment (kbdharun) [reply pages/common/eol.md:3]: Sorry, Thanks for the fix. I made the suggestions pretty late yesterday.

**Review by kbdharun — APPROVED** [review]

**Comment by adriens** [comment]
:clap: :rocket:

**Comment by adriens** [comment]
:crossed_fingers:

File: pages/common/eol.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+# eol
+
+> CLI to show end-of-life dates (EoLs) for a number of products.
```
Comment (acuteenvy) [inline pages/common/eol.md:3]:
```suggestion
> Show end-of-life dates (EoLs) for a number of products.
```

**Review by acuteenvy — COMMENTED** [review]
One more minor suggestion, as most of the tools documented here are for the command-line, I don't think it's necessary to highlight that in every page.

**Review by acuteenvy — APPROVED** [review]

**Comment by adriens** [comment]
:rocket:

**Comment by adriens** [comment]
image

**Comment by adriens** [comment]
:clap: 

- https://github.com/hugovk/norwegianblue/issues/161

**Pull request merged**